File: Java/Program/Cheetah.java

class Cheetah extends Cat {
    private double speed;

    public Cheetah(String gender, double speed) {
        super(gender, "Carnivore", "Cheetah", 8, 10, true);
        this.speed = speed;
    }

    public void display() {
        System.out.println("CHEETAH");
        displayInfo();
    }

    public void displayInfo() {
        super.displayInfo();
        System.out.println("> Speed: " + speed + " km/h");
    }
}

File: Python/Program/Cheetah.py
from Cat import Cat


class Cheetah(Cat):
    def __init__(self, gender, speed):
        super().__init__(gender, "Carnivore", "Cheetah", 8, 10, True)
        self.speed = speed

    def display(self):
        print("CHEETAH")
        self.display_info()

    def display_info(self):
        super().display_info()
        print(f"> Speed: {self.speed} km/h")
